app/views/pages/home.html.erb

```diff
@@ -53,6 +53,28 @@ technical talent shortage. -->
   </div>
   <br><br>
 
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12" role="main">
+        <p class="lead">
+          In the United States over 200,000 computing jobs open annually and 30,000 are filled by Computer Science graduates; 
+          however, that number is expected to rise to 1.2 million by 2020. Meanwhile, we have 250,000 U.S. military personnel that exit the service annually, including some because of sequestration.  Currently, there are almost <a href="http://www.bls.gov/news.release/pdf/vet.pdf">500,000 unemployed veterans</a> with over 70,000 of us between the ages of 25 and 34, which is 
+          the typical age of separation, as of 2015 according to the Bureau of Labor Statistics. 
+          </br>
+          When you join our ranks you'll be joining our growing military veterans coding community. In <a href="http://operation-code.slack.com/">Slack</a>,
+          you'll network, share resources, find job opportunities, get help with your code, and become eligible for conference scholarships and coding 
+          scholarships. Together we are Operation Code.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class='col m12 center'>
```

Comment:
I don't think you can test from git right now. Heroku has a way to do this but I haven't hooked it up yet. 

Can you remove the form render and keep the text? We can file a new ticket on adding a signup form to the front page. It should probably be way simpler too. Maybe just enter your email and get an invite to sign up, set your password etc.